modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjsp

```diff
@@ -19,4 +19,6 @@ DLUtil.getGroupIds(themeDisplay);
 DLUtil.getImagePreviewURL(null, null);
 
 themeDisplay.getPortletGroupId();
+
+portletDisplay.getPortletInstanceConfiguration(AssetPublisherPortletInstanceConfiguration.class);
```

Comment:
@MicaelleSilvaa, for JSP files, this test looks a little different, can you please check the deleted JSP test file on the previous commit and make sure that you're propagating it here as the original test?